darwin/path_utils.py

```diff
@@ -41,3 +44,49 @@ def deconstruct_full_path(filename: str) -> Tuple[str, str]:
     """
     posix_path = PurePosixPath("/") / filename
     return str(posix_path.parent), posix_path.name
+
+
+def parse_manifest(path: Path) -> dict:
+    """
+    Parses the given manifest and returns a list of all the properties in it.
+
+    Parameters
+    ----------
+    path : str
+        The path to the manifest.
+
+    Returns
+    -------
+    list[Property]
+        A list of all the properties in the given manifest.
+    """
+    with open(path) as f:
+        manifest = json.load(f)
+
+    return manifest
```

Comment:
the reason why I put both parsing functions on different files:
- `parse_manifest` in path_utils.py
- `parse_properties` in datatypes.py
is because it needs the `Property` class to be imported from datatypes.py file - and when imported in path_utils.py, it creates a cyclic dependency on both files.

Other solution would be to move the `Property` class to path_utils.py? Thoughts?